software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcess.java

```diff
@@ -103,7 +103,25 @@
     // TODO If automatic updates are enabled and there are updates waiting to be installed, thirty minutes may not be sufficient...
     ConfigKey<Duration> REBOOT_COMPLETED_TIMEOUT = ConfigKeys.newDurationConfigKey("reboot.completed.timeout",
             "duration to wait whilst waiting for a machine to finish rebooting, and thus to become available again", Duration.minutes(30));
-    
-    AttributeSensor<Integer> RDP_PORT = Sensors.newIntegerSensor("rdpPort");
-    AttributeSensor<Integer> WINRM_PORT = Sensors.newIntegerSensor(WinRmTool.PROP_PORT.getName());
+
+    AttributeSensor<Integer> RDP_PORT = Sensors.newIntegerSensor("rdp.port", "RDP port used by the machine of this entity.");
+    AttributeSensor<Integer> WINRM_PORT = Sensors.newIntegerSensor("winrm.port", "WinRM port used by the machine of this entity.");
+
+    /**
+     * @deprecated since 0.9.0; use {@link #RDP_PORT} instead.
+     */
+    @Deprecated
+    AttributeSensor<Integer> RDP_PORT_CAMEL_CASE = Sensors.newIntegerSensor("rdpPort", "[DEPRECATED] instead use 'rdp.port'");
```

Comment:
Can you also add the description parameter in the call to `Sensors.newIntegerSensor`. See my previous comment with the example code-snippet for doing that. Otherwise non-java users (e.g. yaml and web-console users) won't see anything about it being deprecated.